src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -32,10 +49,71 @@ const ChangeRequestsTable: React.FC = () => {
 
   const theme = useTheme();
 
+  useEffect(() => {
+    const handleResize = () => {
+      setWindowSize(window.innerWidth);
+    };
+
+    // Attach the event listener to the window object
+    window.addEventListener('resize', handleResize);
+
+    // Remove the event listener when the component unmounts
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
   if (isLoading || !data) return <LoadingIndicator />;
 
   if (isError) return <ErrorPage message={error?.message} />;
 
+  const smallColumns: GridColDef[] = [
+    {
+      ...baseColDef,
```

Comment:
since all these are the same as the ones below just make them all their own thing and call them twice instead of having the same code twice